frontend/src/pages/files/FileControlsWrapper.tsx

```diff
@@ -296,12 +296,11 @@ const FileManager = observer((props: FileManagerProps) => {
               if (path.type !== 'mine' && path.type != 'private') {
                 return (
                   <>
-                    <fieldset>
                       {(() => {
-                        if (props.cursor.searchJson.getAttribute("notes", "") != "") {
-                          return (
+                        let sidebarContent = []
+                        if (props.cursor.searchJson.getAttribute("notes", "") !== "") {
+                          sidebarContent.push (
                             <>
```

Comment:
These should have key="<something unique>". See https://react.dev/learn/rendering-lists for details on the `key` prop

Reply:
Thank you for pointing me to the documentation. I have set the elements as part of a list and added a key for each element. I set as key the name of the attribute, since it should not be repeated among siblings in the list.